index.ts

```diff
@@ -1,44 +1,34 @@
 import Grid from './src/grid';
-
 import { html } from './src/util/html';
 import { h, createElement, Component, createRef } from 'preact';
-import { useEffect, useRef } from 'preact/hooks';
-import { UserConfig, Config } from './src/config';
-import { BaseComponent, BaseProps } from './src/view/base';
-import {
-  PluginPosition,
-  PluginBaseComponent,
-  PluginBaseProps,
-} from './src/plugin';
-import { BaseActions } from './src/view/base/actions';
+import { useEffect, useRef, useState } from 'preact/hooks';
+import { Config } from './src/config';
+import { PluginPosition } from './src/plugin';
 import { ID } from './src/util/id';
 import { className } from './src/util/className';
 import Row from './src/row';
 import Cell from './src/cell';
-import BaseStore from './src/view/base/store';
-import Dispatcher from './src/util/dispatcher';
+import { useConfig } from './src/hooks/useConfig';
+import { useStore } from './src/hooks/useStore';
+import useSelector from './src/hooks/useSelector';
 
 export {
   Grid,
   ID,
-  Dispatcher,
   Row,
   Cell,
-  BaseActions,
-  BaseStore,
   className,
   html,
-  UserConfig,
   Config,
-  BaseComponent,
-  BaseProps,
   PluginPosition,
-  PluginBaseComponent,
-  PluginBaseProps,
   h,
   createElement,
   Component,
   createRef,
   useEffect,
-  useRef
+  useRef,
```

Comment:
Why do we re-export preact's exports? I'm curious!

Reply:
That's mainly for people who want to extend Grid.js or write Grid.js components. Note that when we compile Grid.js, Preact is already included in the output (i.e you don't have to install Preact separately).